### ISSUE TITLE
Login page still available

### ISSUE THREAD

**Issue by rheaplex** [cc-archive/wiki#19]
https://wiki.creativecommons.org/wiki/Special:UserLogin

is still offering to let me log in "as a different user" with non-CCID credentials.

**Comment by rheaplex** [comment]
Vector login button links to: http://mediawiki.local/index.php?title=Special:UserLogin&returnto=Main+Page

This sends us to: https://login.creativecommons.org//login?service=http://mediawiki.local/index.php?title=Special:UserLogin&returnto=Main+Page

Bootstrap theme button links to: https://wiki.creativecommons.org/wiki/Special:UserLogin

This sends us to: https://login.creativecommons.org//login?service=https://wiki.creativecommons.org/wiki/Special:UserLogin

Investigating theme differences.

**Comment by rheaplex** [comment]
I can see what needs doing. Will finish up tomorrow.

**Comment by rheaplex** [comment]
Initial login redirect made more robust in - https://github.com/creativecommons/bootstrap-mediawiki/commit/700dfa4165cc66f38a1f2b7f8872c0eaaa1aabe8

The availability of Special:UserLogin after logging in with CAS is expected behaviour.

I've tried to reduce confusion on the Login page as much as possible by editing the fields listed on:

https://wiki.creativecommons.org/wiki/Special:UserLogin?uselang=qqx

this will haunt us if we move back from CASAuth, so I am making a note of that here.

**Comment by rheaplex** [comment]
I've disabled the login form on Special:UserLogin by adding an extension to our theme. See here:

https://github.com/creativecommons/bootstrap-mediawiki/commit/32f10362ee481f6e39e470ea482810b1a2b4f5a2

And here for instructions on how to activate it:

https://github.com/creativecommons/bootstrap-mediawiki/blob/master/NoUserLoginForm.php

I've tested login/logout and access to the page when logged in/out locally and on wiki.cc.org.

CCID-only login is now robust.

 @mattl if this is too terribly wrong let me know and I'll separate it out into its own project.
